README.md

```diff
@@ -71,4 +118,8 @@ work in the future as more strict token checks are implemented in DeviceManager.
 
 ## How to use
 
-There are a few examples on how to use DeviceManager in [this page](./docs/using-device-manager.rst).
+There are a few examples on how to use DeviceManager in [Device Manager Documentation](https://dojotdocs.readthedocs.io/projects/DeviceManager/en/latest/using-device-manager.html).
+
+## API Documentation
+
+URL to api documentation https://dojot.github.io/device-manager/apiary_latest.html for development and latest version of this Device Manager.
```

Comment:
I think the link to development was missing

Reply:
Thanks, I forgot to add the link. I'll add.